tools/cargo_bazel/src/annotation.rs

```diff
@@ -181,10 +182,29 @@ impl LockfileAnnotation {
 
         // Handle any git repositories
         if let Some(git_ref) = source.git_reference() {
+            let mut manifest_dir = pkg.manifest_path.clone();
+            manifest_dir.pop();
+            let mut ancestors = manifest_dir.ancestors();
+            let mut git_root = None;
+            while let Some(ancestor) = ancestors.next() {
+                if ancestor.join(".git").is_dir() {
+                    git_root = Some(ancestor);
+                    break;
+                }
+            }
+            let git_root = git_root.unwrap_or_else(|| panic!("Expected git repository to have a .git directory for package {} with manifest {}", pkg.name, pkg.manifest_path));
+
+            let strip_prefix = if git_root == manifest_dir {
+                None
+            } else {
+                Some(manifest_dir.strip_prefix(git_root).unwrap().to_string())
```

Comment:
Do you know if this will have backslashes on windows?

Reply:
It will! Fixed!